components/custom-button/custom-button.style.js

```diff
@@ -47,6 +47,7 @@ const selectButton = (props) => {
 
   return props.inverted ? InvertedButtonStles : BaseButtonStyles;
 };
+// console.log(selectButton(props));
```

Comment:
Can we remove this please ??

Reply:
Thanks for pointing it out. Made the changes